### PR TITLE
bpo-44605: Teach @total_ordering() to work with metaclasses

### PR DESCRIPTION
**Pull request by rhettinger**
https://bugs.python.org/issue44605

### CODE REVIEW

**Review by ambv — APPROVED** [review]

**Comment by pablogsal** [comment]
@rhettinger Please, make sure you get at least a review from another core Dev before landing changes in the release candidate. I send several communications about this, the last one in this email:

https://mail.python.org/archives/list/python-dev@python.org/thread/RLO5Y6FPOPUJAJVN6MHRIKUSMSRUXUT6/

Here is the important extract:

>Once the 3.10 branch reaches RC status, it only can have bug fixes applied that have been reviewed by other core developers (so you cannot merge your own PR without review even if you are a core dev). Generally, these issues must be severe enough (e.g. crashes) that they deserve fixing before the final release. All other issues should be deferred to the next development cycle (Python 3.10.1) since stability is the strongest concern at this point. Also bear in mind that once we reach the RC, the *ABI is frozen* and cannot change even for bug fixes.
>While the goal is to have no code changes between an RC and a final release, there may be a need for final documentation o test fixes. Any such proposed changes should be discussed first with the release manager.
>*You cannot skip the peer review during an RC*, no matter how small! Even if it is a simple copy-and-paste change, everything requires peer review from a core developer

**Comment by pablogsal** [comment]
My last message is in reference to the backport, not to this PR.

**Pull request merged**
